src/Sentry.Unity/WebGL/SentryWebGL.cs

```diff
@@ -0,0 +1,227 @@
+using System;
+using System.Collections;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Sentry.Extensibility;
+using Sentry.Http;
+using Sentry.Infrastructure;
+using Sentry.Internal;
+using Sentry.Internal.Http;
+using Sentry.Protocol.Envelopes;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace Sentry.Unity.WebGL
+{
+    /// <summary>
+    /// Configure Sentry for WebGL
+    /// </summary>
+    public static class SentryWebGL
+    {
+        /// <summary>
+        /// Configures the WebGL support.
+        /// </summary>
+        /// <param name="options">The Sentry Unity options to use.</param>
+        public static void Configure(SentryUnityOptions options)
+        {
+            options.DiagnosticLogger?.LogDebug("Updating configuration for Unity WebGL.");
+
+            // Caching transport relies on a background thread
+            options.CacheDirectoryPath = null;
+            // Note: we need to use a custom background worker which actually doesn't work in the background
+            // because Unity doesn't support async (multithreading) yet. This may change in the future so let's watch
+            // https://docs.unity3d.com/2019.4/Documentation/ScriptReference/PlayerSettings.WebGL-threadsSupport.html
+            options.BackgroundWorker = new WebBackgroundWorker(options, SentryMonoBehaviour.Instance);
+
+            options.CrashedLastRun = () => false; // no way to recognize crashes in WebGL yet
+
+            // Still can't find out what's using Threads so:
```

Comment:
I recall writing this one :D
I'm not sure which of the properties below are needed

Reply:
none of them has to be disabled - at least I didn't find any issues with my manual testing and don't really look like having any problematic code (a.k.a. task await). Though I must say I didn't know how to trigger some of the features, like the`AppDomainUnhandledExceptionCapture`.